doc/opuschgain.1

```diff
@@ -0,0 +1,47 @@
+.\" This manpage has been automatically generated by docbook2man 
+.\" from a DocBook document.  This tool can be found at:
+.\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
+.\" Please send any bug reports, improvements, comments, patches, 
+.\" etc. to Steve Cheng <steve@ggi-project.org>.
+.TH "OPUSCHGAIN" "1" "2019-05-25" "1.5.4" "User Manual"
+
+.SH NAME
+opuschgain \- Updated Ogg Opus file output gain and R128 gain tag
+
+.SH DESCRIPTION
+.PP
+opuschgain edits the output gain of Ogg Opus file. If the \fIsrcfile\fR has the \fIR128_TRACK_GAIN\fR \fIR128_ALBUM_GAIN\fR tag, update it together. See \fIRFC 7845 5.2 5.2.1.\fR for information on these two tags.
+
+.SH SYNOPSIS
+\fBopuschgain\fR [ \fB-i \fIidx\fB\fR ] { \fB-g \fIgain\fB\fR | \fB-s \fIscale\fB\fR | \fB-0\fR } [ \fB-1Qr\fR ] \fB\fIsrcfile\fB\fR [ \fB\fIoutput\fB\fR ]
+
```

Comment:
Synopsis and description are swapped. see `man 7 man-pages`.

Reply:
Hmm. Really. Update.

Reply:
Updated.